src/jvm/test/org/jetbrains/kotlinx/lincheck/test/verifier/durable/detectable/LogQueueTest.kt

```diff
@@ -193,8 +193,8 @@ internal class LogQueue(threadsNumber: Int) : RecoverableQueue {
     }
 }
 
-internal abstract class LogQueueFailingTest :
-    AbstractNVMLincheckFailingTest(Recover.DETECTABLE_EXECUTION, THREADS_NUMBER, SequentialQueue::class) {
+internal abstract class LogQueueFailingTest //: AbstractNVMLincheckFailingTest(Recover.DETECTABLE_EXECUTION, THREADS_NUMBER, SequentialQueue::class)
```

Comment:
Is it correct?

Reply:
Log queue algorithm is not ready for now, so it is muted.